data/persons.txt

```diff
@@ -768,3 +768,16 @@ ship "Ursa Polaris"
 	explode "big explosion" 40
 	explode "huge explosion" 50
 
+
+person "Subsidurial"
+	government "Indigenous Lifeform"
+	frequency 100
+	personality
+		timid unconstrained coward appeasing uninterested mining harvests mute
+	ship "Subsidurial"
+		"never disabled"
+	system
+		neighbor government "uninhabited"
+		neighbor government "Wanderer"
+		neighbor government "Remnant"
```

Comment:
`system` is a LocationFilter, which means it will work the same way as a `source` or `destination`, which also means that this is looking for a system that is a neighbor to Uninhabited, Wanderer, AND Remnant. 
`neighbor government "uninhabited" "Wanderer" "Remnant` is what you're looking for.

Reply:
Fixed. Thanks!